powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/filter/PowerAuthRequestFilterBase.java

```diff
@@ -60,7 +60,7 @@ public static ResettableStreamHttpServletRequest filterRequest(HttpServletReques
             // Parse the query parameters
             String queryString = httpRequest.getQueryString();
 
-            if (queryString != null && queryString.length() > 0) {
+            if (queryString != null && !queryString.isEmpty()) {
```

Comment:
Suggestion: we can also use `StringUtils.hasText`: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/StringUtils.html#hasText(java.lang.String)

Reply:
Here, we should use `StringUtils.hasLength(str)` as a query parameter consisting just of whitespaces should go into the signature.